template/src/constants/index.js

```diff
@@ -1 +1,4 @@
 export const TOKEN_KEY = 'TOKEN'
+export const REFRESH_TOKEN_KEY = 'REFRESH_TOKEN_KEY'
+export const RESPONSE_SUCCESS = [200, 201]
+export const UNAUTHORIZED = [401, 403]
```

Comment:
The case 500 server response HTML instead of message string error, this case only appear for some projects. So I feel we don't need to support this case

Reply:
I think we still need cover this status code by return empty string instead of HTML first then when we using this template we could change that message later

Reply:
We can implement other codes in real project, i'll make this resolve

Reply:
Please tell me more about other codes in real project